src/Web/Sfa.Das.Sas.Shared.Components.Web/Startup.cs

```diff
@@ -29,15 +31,22 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
+            services.AddMemoryCache();
 
             var fatConfig = new FatSharedComponentsConfiguration();
             Configuration.Bind("fatSharedComponents", fatConfig);
             services.AddSingleton<IFatConfigurationSettings>(fs => fatConfig);
 
+            services.AddSingleton<ICacheStorageService, CacheStorageService>();
+            services.AddScoped<TrainingProviderDetailQueryViewModel>();
```

Comment:
these shouldnt be needed here at all first is included in the AddFatSharedComponentsCaching call, im not sure what the second is doing or for!